src/backend/docker-compose.env

```diff
@@ -1,28 +1,24 @@
 DJANGO_ENV=development
 DEBUG=1
 DJANGO_ALLOWED_HOSTS="www.example.com localhost 127.0.0.1 *"
-
 # DB_ENGINE=django.db.backends.postgresql_psycopg2
 # DB_DATABASE=CoreAPI
 # DB_USER=admin
 # DB_PASSWORD=admin
 # DB_PORT=5432
-
 USING_REDIS_QUEUE=1
 REDIS_QUEUE_LOCATION=redis
-
 PYTHONUNBUFFERED=1
-
 RS_URL="http://recommender_system:8086"
-
 NOTIFICATIONS_CONFIG_PATH="./config/notifications.json"
-
 EMAIL_USE_SSL=1
 EMAIL_PORT=465
 EMAIL_HOST=smtp.seznam.cz
 EMAIL_HOST_USER=ecoseller-storefront@email.cz
-EMAIL_HOST_PASSWORD="yourpassword # note: after you change this line, please run `git update-index --assume-unchanged .\src\backend\docker-compose.env`"
+# note: after you change this line, please run `git update-index --assume-unchanged .\src\backend\docker-compose.env`
+EMAIL_HOST_PASSWORD=malastrana25
 EMAIL_FROM=Storefront<ecoseller-storefront@email.cz>
 USE_ELASTIC=1
 ELASTIC_HOST="elasticsearch:9200"
 ELASTIC_AUTO_REBUILD_INDEX=0
+USING_REDIS_QUEUE=1
```

Comment:
`USE_REDIS_QUEUE` is already present on line 11

Reply:
Removed.